README.md

```diff
@@ -211,15 +223,18 @@ git submodule update --init --recursive --jobs 0
 ```
 
 #### Install PyTorch
-On Linux
+**On Linux**
+
+If you're compiling for AMD ROCm then first run this command:
 ```bash
-export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
-python setup.py install
+# Only run this if you're compiling for ROCm
+python tools/amd_build/build_amd.py
 ```
 
-Note that if you are compiling for ROCm, you must run this command first:
+Install PyTorch
 ```bash
-python tools/amd_build/build_amd.py
+export CMAKE_PREFIX_PATH=${CONDA_PREFIX:-"$(dirname $(which conda))/../"}
```

Comment:
Is this really necessary? I never had to pass this option

Reply:
No idea. This was the pre-existing instruction and I didn't want to modify it right now in case there are corner cases where this is needed.